tests/test-customize-changesets-controller.php

```diff
@@ -90,6 +354,161 @@ public function test_create_item() {
 	}
 
 	/**
+	 * Test tha creating changeset with invalid data fails.
+	 */
+	public function test_create_item_invalid_data() {
+		wp_set_current_user( self::$editor_id );
+
+		$request = new WP_REST_Request( 'POST', '/customize/v1/changesets' );
+		$request->set_body_params( array(
+			'customize_changeset_data' => '[MALFORMED]',
+		) );
+		$response = $this->server->dispatch( $request );
+
+		$this->assertErrorResponse( 'invalid_customize_changeset_data', $response, 400 );
+	}
+
+	/**
+	 * Test that editor can create a changeset.
+	 */
+	public function test_create_item_as_editor() {
+		wp_set_current_user( self::$editor_id );
+
+		$request = new WP_REST_Request( 'POST', '/customize/v1/changesets' );
+		$request->set_body_params( array(
+			'customize_changeset_data' => array(
```

Comment:
I think that `customize_changeset_data` should be just `settings` to mirror the property being returned in the response.